tests/tck/features/lookup/LookUpLimit.feature

```diff
@@ -0,0 +1,175 @@
+      | "Kyrie Irving"      | "LeBron James"      | 0       |
+      | "LaMarcus Aldridge" | "Tim Duncan"        | 0       |
+      | "LaMarcus Aldridge" | "Tony Parker"       | 0       |
+      | "Marco Belinelli"   | "Tim Duncan"        | 0       |
+      | "Marco Belinelli"   | "Tony Parker"       | 0       |
+      | "Rajon Rondo"       | "Ray Allen"         | 0       |
+      | "Ray Allen"         | "Rajon Rondo"       | 0       |
+      | "Rudy Gay"          | "LaMarcus Aldridge" | 0       |
+    And the execution plan should be:
+      | id | name              | dependencies | operator info  |
+      | 4  | DataCollect       | 5            |                |
+      | 5  | Sort              | 6            |                |
+      | 6  | Project           | 7            |                |
+      | 7  | EdgeIndexFullScan | 0            | {"limit": "2"} |
+      | 0  | Start             |              |                |
+    When profiling query:
+      """
+      LOOKUP ON player WHERE player.age == 33 Limit 2 | ORDER BY $-.VertexID
+      """
+    Then the result should be, in any order:
+      | VertexID            |
+      | "Chris Paul"        |
+      | "Dwight Howard"     |
+      | "LaMarcus Aldridge" |
+      | "Rajon Rondo"       |
+    And the execution plan should be:
+      | id | name               | dependencies | operator info  |
+      | 4  | DataCollect        | 5            |                |
+      | 5  | Sort               | 6            |                |
+      | 6  | Project            | 7            |                |
+      | 7  | TagIndexPrefixScan | 0            | {"limit": "2"} |
+      | 0  | Start              |              |                |
+    When profiling query:
+      """
+      LOOKUP ON like WHERE like.likeness == 90 Limit 2 | ORDER BY $-.SrcVID
+      """
+    Then the result should be, in any order:
+      | SrcVID              | DstVID               | Ranking |
+      | "Amar'e Stoudemire" | "Steve Nash"         | 0       |
+      | "Carmelo Anthony"   | "Chris Paul"         | 0       |
+      | "Chris Paul"        | "Carmelo Anthony"    | 0       |
+      | "Chris Paul"        | "Dwyane Wade"        | 0       |
+      | "Dwyane Wade"       | "Carmelo Anthony"    | 0       |
+      | "Dwyane Wade"       | "Chris Paul"         | 0       |
+      | "Jason Kidd"        | "Steve Nash"         | 0       |
+      | "Klay Thompson"     | "Stephen Curry"      | 0       |
+      | "Luka Doncic"       | "Dirk Nowitzki"      | 0       |
+      | "Luka Doncic"       | "Kristaps Porzingis" | 0       |
+      | "Paul Gasol"        | "Kobe Bryant"        | 0       |
+    And the execution plan should be:
+      | id | name                | dependencies | operator info  |
+      | 4  | DataCollect         | 5            |                |
+      | 5  | Sort                | 6            |                |
+      | 6  | Project             | 7            |                |
```

Comment:
The limit is missing in the graph layer

Reply:
> The limit is missing in the graph layer

I think ok, limit node should not appear in the graph layer for this query. WDYT?
`lookup on tag where .. limit | limit` should appear both storage and graph layer.

Reply:
The limit clause of lookup is different with individual limit sentence. It means limit by partition.